components/Chips/src/MDCChipView.m

```diff
@@ -569,6 +589,7 @@ - (void)layoutSubviews {
   [super layoutSubviews];
 
   _inkView.frame = self.bounds;
+  _rippleView.frame = self.bounds;
```

Comment:
This shouldn't be necessary of the frame is corrected in `setEnableRippleBehavior`.

Reply:
It is necessary and is aligned with how inkView is set up, and the frame isn't corrected in the setter, but rather here.

Reply:
The frame could be fixed in the setter rather than doing the layout work twice here.  I think that's a good improvement over what was done for Ink.